client/src/components/Related/ComparisonModal.jsx

```diff
@@ -15,7 +15,7 @@ const ComparisonModal = (props) => {
       onClick={closeModal}>
       {/* <button onClick={closeModal}>X</button> */}
```

Comment:
saving or testing with this?

Reply:
oh i'll get rid of that real quick, i couldn't get the close button to work but it doesn't need it

Reply:
fixed!